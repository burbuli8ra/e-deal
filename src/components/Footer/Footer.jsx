import React from 'react';
import Styled from './Footer.style';

const Footer = () => {
  const currentYear = new Date().getFullYear();

  return (
    <Styled.Footer data-testid="footer">
      {currentYear} Â© eDeal
    </Styled.Footer>
  );
};

export default Footer;