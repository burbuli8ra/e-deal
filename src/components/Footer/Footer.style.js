import styled from '@emotion/styled';
import { colors } from 'theme';

const Footer = styled.footer`
  background-color: ${colors.white};
  box-shadow: 0 0 9px 0 rgba(140, 140, 158, 0.25);
  color: ${colors.darkPastelBlue};
  font-size: 14px;
  padding: 20px;
  text-align: center;
`;
Footer.displayName = 'Footer';

export default {
  Footer
};