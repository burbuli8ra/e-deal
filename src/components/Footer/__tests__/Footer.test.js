import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../Footer';

const setup = props => <Footer {...props} />;

describe('Footer component', () => {
  let props;
  let wrapper;

  beforeEach(() => {
    props = {};
    wrapper = render(setup(props));
  });

  test('should match snapshot', () => {
    const { asFragment } = wrapper;
    expect(asFragment()).toMatchSnapshot();
  });
});