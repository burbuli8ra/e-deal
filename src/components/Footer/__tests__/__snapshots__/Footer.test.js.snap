// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `
<DocumentFragment>
  <footer
    class="css-1ydo3a8"
    data-testid="footer"
  >
    2020 Â© eDeal
  </footer>
</DocumentFragment>
`;
